reference-data-proxy/src/cronJobs/eStoreDealFolderCreationJob.cron.ts

```diff
@@ -28,9 +25,6 @@ export const eStoreDealFolderCreationJob = async (eStoreData: Estore) => {
       // stop and the delete the cron job to release the memory
       eStoreCronJobManager.deleteJob(`Deal${eStoreData.dealId}`);
 
-      // update the `tfm-deals` collection once the buyer and deal folders have been created
-      tfmDealsCollection.updateOne({ _id: ObjectId(eStoreData.dealId) }, { $set: { 'tfm.estore': { siteName: eStoreData.siteName } } });
```

Comment:
Are we not going to update the collection when deal folder is created?

Reply:
I've moved it earlier on in the process. it's in this file:
`reference-data-proxy/src/cronJobs/eStoreTermStoreAndBuyerFolder.cron.ts`